unison-src/transcripts-round-trip/main.md

```diff
@@ -7,23 +7,22 @@ This transcript verifies that the pretty-printer produces code that can be succe
 
 ## How to use this transcript: checking round-trip for inline definitions
 
-```unison:hide
+```unison:hide roundtrip.u
```

Comment:
I did some cleanup in this file, sending the updates / edits to `roundtrip.u` to avoid clobbering your local `scratch.u` if you're running this transcript locally.